apps/labs/posts/unlocking-c-level-performance-in-df-apply.md

```diff
@@ -0,0 +1,463 @@
+---
+title: 'Unlocking C-level performance in pandas DataFrame.apply with numba'
+authors: [thomas-li]
+published: October 15, 2023
+description: 'A high-level overview of the new numba engine in DataFrame.apply'
+category: [PyData ecosystem]
+featuredImage:
+  src: /posts/numpy-python-api-cleanup/numpy-python-api-cleanup-featured.png
```

Comment:
I should generate an image for this.

@MarcoGorelli Is there any existing pandas image I can re-use?

Reply:
I added two suggestions to generate a graph/graphic. Maybe a reworked version of one of those could serve as the header image too.